### DISCUSSION TITLE
Dual Winsinn 5015 24V part cooling fans PWM settings

### DISCUSSION THREAD

**Discussion by victorwennerholm** [mriscoc/Ender3V2S1#712] [Q&A]
This firmware is great! I only have one issue left to solve (been googling like crazy, but none of the suggested setups seem to work)

I have dual winsinn 5015 24V fans installed and they move A LOT of air. However, if I turn them lower than 100% an irritating oscillating noise starts. I've tried adjusting these parameters in different combinations and compiling it and this changes the noise, but I haven't found a perfect setup yet... Anyone with the same setup that's found a magic combination?

FAN_SOFT_PWM - defined
SOFT_PWM_SCALE - 0 to 4
SOFT_PWM_DITHER - defined as well as undefined

FAST_PWM_FAN - defined as well as undefined
FAST_PWM_FAN_FREQUENCY - 31400 / 39200

**Comment by victorwennerholm** [comment]
Oh, I should've probably mentioned that - the machine is an Ender 3 v2 with a 4.2.7 stock motherboard.

**Reply by mriscoc** [reply]
This firmware is based on the Marlin firmware, maybe you can search the Marlin repository to see if anyone has similar fans.
